client/src/unifyfs.c

```diff
@@ -654,6 +654,8 @@ static int process_read_data(read_req_t* read_reqs, int count, int* done)
     /* get number of read replies in shared memory */
     size_t num = shm_hdr->meta_cnt;
 
+    LOGDBG("received %lu response from server", num);
```

Comment:
"%zu" for size_t

Reply:
Thanks @MichaelBrim . I have fixed other places where I made the same error.